dictionary-std/src/uids.rs

```diff
@@ -1420,10 +1420,10 @@ pub const IMPLANT_TEMPLATE_GROUP_INFORMATION_MODEL_MOVE: &str = "1.2.840.10008.5
 /// SOP Class: Implant Template Group Information Model - GET
 #[rustfmt::skip]
 pub const IMPLANT_TEMPLATE_GROUP_INFORMATION_MODEL_GET: &str = "1.2.840.10008.5.1.4.45.4";
-/// Application Hosting Modle: Native DICOM Model
+/// Application Hosting Model: Native DICOM Model
```

Comment:
Looks like the typo leaked into the generated files. It's fine to fix them here, might also think about generating updated files soon.